# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../iwlwifi/KVER.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- iwlwifi-0.0.11/Makefile	2007-03-05 21:22:24.000000000 +0100
+++ iwlwifi-0.0.11/new	2007-04-09 12:47:13.000000000 +0200
@@ -30,7 +30,11 @@
 #        debug in any distribution packages. - James
 EXTRA_CFLAGS += -DCONFIG_IWLWIFI_DEBUG=y
 
-export MAC80211_INC ?= /lib/modules/$(shell uname -r)/source/include/net
+# KSRC should be set to the path to your sources
+# modules are installed into KMISC
+KVER  := $(shell uname -r)
+
+export MAC80211_INC ?= /lib/modules/$(KVER)/source/include/net
 EXTRA_CFLAGS += -I$(MAC80211_INC)
 
 ifdef KBUILD_EXTMOD
@@ -69,7 +73,6 @@
 
 # KSRC should be set to the path to your sources
 # modules are installed into KMISC
-KVER  := $(shell uname -r)
 # KSRC_OUTPUT should be overridden if you are using a 2.6 kernel that
 # has it's output sent elsewhere via KBUILD_OUTPUT= or O=
 #
